### DISCUSSION TITLE
Welcome to sd-webui-traintrain Discussions!

### DISCUSSION THREAD

**Discussion by hako-mikan** [hako-mikan/sd-webui-traintrain#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by muooon** [comment]
Thank you hako-mikan and everyone to TrainTrain.
When I started using SD15, it was Ryzen2400G. I started generating images using just the CPU, and now I'm playing SDXL with a 12700F and RTX3600-12G.
I would like to continue creating LoRA with TrainTrain and have fun with it.
I'm interested in Nvidia's "Sana," which was announced the other day, as it seems to be a resource-saving model.
I hope that with the development of TrainTrain, it will be possible to perform FFT on XL models as well.
It's easy for anyone to get started - this is why I like it the most.
---
hako-mikanさん 、そして皆さん、TrainTrainに感謝します。
SD15を触り始めたときはRyzen2400Gでした。
CPUだけで画像生成を開始し、今は12700FとRTX3600-12GでSDXLを遊んでいます。
今後もTrainTrainでLoRAをつくり楽しみたいと思っています。
先日発表された Nvidiaの"Sana"は省資源のモデルのようで関心を持ってます。
TrainTrainの発展でXLのモデルもFFTできるようになるといいな、と思っています。
誰でも簡単に始められる－ここが最も気に入ってる理由です。
